### DISCUSSION TITLE
Can vitest be used for server-side node application testing?

### DISCUSSION THREAD

**Discussion by pauldotknopf** [vitest-dev/vitest#5619] [General]
The reason I ask is because I'd like to introduce testing into my org.

We have browser-based applications that are using vite, which makes vitest and valid choice.

However, we also have a few server applications, and I'd have to have multiple test frameworks in our organization.

**Comment by hzamir** [comment]
I'm also curious on this point, please share what you find.

**Comment by hi-ogawa** [comment]
You should be able to use Vitest for any javascript testing, just like you use Jest, Mocha, or Node test runner.
If you have any specific concern, please elaborate more details, for example, which server framework / library you use etc...
